package.json

```diff
@@ -32,6 +33,7 @@
   "license": "MIT",
   "devDependencies": {
     "@glennsl/bs-jest": "^0.5.1",
+    "bisect_ppx": "^2.3.1",
```

Comment:
bisect_ppx has to be a regular dep; see https://github.com/aantron/bisect_ppx/issues/309#issuecomment-616076931. This is because the ppx-flags in `bsconfig.json` cause Bisect to always be called. It's just when `BISECT_ENABLE=yes` is missing, Bisect leaves the code clean and uninstrumented. But the code still gets passed through Bisect.

Once BuckleScript releases dev `ppx-flags` (https://github.com/BuckleScript/bucklescript/issues/3761) in the next release, it will be correct to move `bisect_ppx` to `devDependencies`, which is where it properly belongs :)